pom.xml

```diff
@@ -41,7 +41,7 @@
     <maven.compiler.target>1.8</maven.compiler.target>
 
     <!-- JBoss Tools Integration Stack version -->
-    <jbtis.version>4.4.1.Final</jbtis.version>
+    <jbtis.version>4.4.2.AM1</jbtis.version>
```

Comment:
Yeah, but we need to test asap with that new tp version as code freeze is next week and if our tooling doesn't work with that TP we got a problem.